src/components/helper.ts

```diff
@@ -0,0 +1,29 @@
+import { Todo } from '../types/Todo';
+import { ForFilteredTodos } from './TodoFilter';
+
+export const forFilteredTodos = (
+  todos: Todo[],
+  query: string,
+  selectedFilter: ForFilteredTodos,
+) => {
+  let preparedTodo = [...todos];
```

Comment:
The `filter` method returns a copy of the array, so you don't need to do it manually.